apps/docs/components/docs/components/code-demo/utils.ts

```diff
@@ -2,7 +2,7 @@ import {get} from "lodash";
 
 import {FileCode} from "./types";
 
-const importRegex = /^(import)\s(?!type(of\s|\s)(?!from)).*?$/gm;
+const importRegex = /^(import\s+(?!type\s+\{)[\s\S]*?;)/gm;
```

Comment:
`(?!type(of\s|\s)(?!from))` change to `(?!type\s+\{)` why need `\{`

Reply:
cover `import type { Selection } from "@nextui-org/react";` case